### PR TITLE
10 functions

### PR DESCRIPTION
**Pull request by ethteck**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀 This PR matches 10 functions (+0.12%) on `us`.

✅ This PR fixes 2 warnings!

File: src/audio/2e230_len_2190.c [hunk 1]
```diff
@@ -820,7 +820,39 @@ void au_load_PRG(AuGlobals* arg0, s32 romAddr) {
...
+    s32 ret = 0;
+    s32 i;
+
+    while (TRUE) {
+        if (song->bgmFileIndex == 0xFFFF) {
+            return ret;
+        }
+
+        if (song->bgmFileIndex == arg0) {
+            for (i = 0 ; i < ARRAY_COUNT(song->bkFileIndex); i++) {
```
Comment (nanaian) [inline src/audio/2e230_len_2190.c:835]:
```suggestion
            for (i = 0; i < ARRAY_COUNT(song->bkFileIndex); i++) {
```

File: src/world/area_kmr/kmr_20/8EBE50.c [hunk 2]
```diff
@@ -32,14 +43,164 @@ ApiStatus func_80240358_8EC178(Evt* script, s32 isInitialCall) {
...
+
+    if (isInitialCall) {
+        script->functionTemp[0] = 0;
+    }
+
+    switch (script->functionTemp[0]) {
+        case 0:
+            numEntries = 0;
+
+            for (i = 0; i < 12; i++) {
```
Comment (nanaian) [inline src/world/area_kmr/kmr_20/8EBE50.c:84]: ARRAY_COUNT(D_80244D24_8F0B44)?

File: src/world/area_kmr/kmr_20/8EDFC0.c [hunk 3]
```diff
@@ -146,8 +146,35 @@ void func_80242A04_8EE824(void) {
...
+    count = 0;
+    for (i = 0; i < ARRAY_COUNT(playerData->badges); i++) {
+        if (playerData->badges[i] != 0) {
+            count++;
+        }
+    }
+    arg0->equippedBadges = count;
+
+    count = 0;
+    for (i = 0; ; i++) {
```
Comment (nanaian) [inline src/world/area_kmr/kmr_20/8EDFC0.c:163]: Better to make this cond explicit I think
```suggestion
    for (i = 0; TRUE; i++) {
```

**Pull request merged**